Manual-Configuration-Steps.md

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
You might want to see the resulting file in the fork where I've done the modifications: https://github.com/jmprieur/active-directory-dotnet-daemon-certificate-credential